.pre-commit-config.yaml

```
@@ -13,24 +13,14 @@ repos:
           - --hang-closing
           - --max-line-length=99
 
-  - repo: https://github.com/PyCQA/isort
-    rev: "5.13.2"
+  - repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.8.6
     hooks:
-      - id: isort
-
-  - repo: https://github.com/pycqa/flake8
-    rev: "7.1.1"
-    hooks:
-      - id: flake8
+      - id: ruff
```

```suggestion
      - id: ruff-format
```
Probably only want to deal with formatting as part of this PR?